rollup.config.js

```diff
@@ -1,24 +1,47 @@
-import {terser} from "rollup-plugin-terser";
+import babel from "@rollup/plugin-babel";
 import filesize from "rollup-plugin-filesize";
+import postcss from "rollup-plugin-postcss"
 import sourcemaps from "rollup-plugin-sourcemaps";
-import babel from "@rollup/plugin-babel";
+import {terser} from "rollup-plugin-terser";
 
-export default {
+export default [{
+    input: "src/less/material.less",
+    output: {
+        dir: "dist/css",
+    },
+    plugins: [
+        postcss({
+            inject: false,
+            extract: "material.css",
+            sourceMap: false,
+            minimize: true,
+        }),
```

Comment:
`postcss` is a good addition!  It was not possible with the old babel-based build but is quite simple with rollup